contracts/Scarb.toml

```diff
@@ -15,4 +15,8 @@ test = "cairo-test --starknet ."
 # quaireaux = { git = "https://github.com/keep-starknet-strange/quaireaux.git" }
 
 [[target.starknet-contract]]
-
+sierra = true
```

Comment:
I'm guessing you're using scarb 0.2 (so not the version here https://github.com/smartcontractkit/chainlink-starknet/pull/224) since these options aren't available until scarb 0.2, which is not compatible with Starknet mainnet/testnet. 

I was talking to @archseer about this, but are we all operating under the assumption that Starknet will update their version to be compatible with cairo alpha-7 by launch date? How confident are we about this? Cause if they don't, we're compiling code and writing tests for code that won't be supported on mainnet.